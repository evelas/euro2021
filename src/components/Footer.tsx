import React from 'react'
import footer from '../assets/img/mainfooter.png';

const Footer: React.FC = () => {
  return (
    <footer className="footer">
      <img src={footer} alt='footer' width="100%" height="100%" />
    </footer>
  )
}

export default Footer
