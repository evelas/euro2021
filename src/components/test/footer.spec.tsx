import { shallow, ShallowWrapper, render } from "enzyme";
import React from "react";
import Footer from "../Footer";

const setUp = () => shallow(<Footer />);

describe("Footer component", () => {
  let component: ShallowWrapper;
  beforeEach(() => {
    component = setUp();
  });

  it("should contain .footer wrapper", () => {
    const wrapper = component.find(".footer");
    expect(wrapper.length).toBe(1);
  });
  it("should render Footer component", () => {
    const component = render(<Footer />);
    expect(component).toMatchSnapshot();
  });
});
