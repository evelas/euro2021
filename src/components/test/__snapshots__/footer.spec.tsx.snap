// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should render Footer component 1`] = `
<footer
  class="footer"
>
  <img
    alt="footer"
    height="100%"
    src="test-file-stub"
    width="100%"
  />
</footer>
`;
